app/models/innovation.rb

```diff
@@ -0,0 +1,11 @@
+class Innovation < ApplicationRecord
+  self.abstract_class = true
+  has_paper_trail
+
+  belongs_to :user, optional: true
+  has_many :category_practices, as: :innovable, dependent: :destroy, autosave: true
+  has_many :categories, through: :category_practices
```

Comment:
We are likely going to need the polymorphic treatment on `PracticePartners` and `PracticeAward` as well. See Figma discussion re: various IPM partners: https://www.figma.com/design/D16fyBygVx88HpyczTuqXs?node-id=2957-15614#916902746